src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -127,8 +146,111 @@ private function fixFunction(Tokens $tokens, $startFunctionIndex)
 
             if ($token->equals(',')) {
                 $this->fixSpace2($tokens, $index);
+                if (!$isMultiline && $this->isNewline($tokens[$index + 1])) {
+                    $isMultiline = true;
+                }
             }
         }
+
+        return $isMultiline;
+    }
+
+    private function ensureFunctionFullyMultiline(Tokens $tokens, $startFunctionIndex)
+    {
+        $endFunctionIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startFunctionIndex);
+        // find out what the indentation is
+        $searchIndex = $startFunctionIndex;
+        do {
+            $prevWhitespaceTokenIndex = $tokens->getPrevTokenOfKind(
+                $searchIndex,
+                [[T_WHITESPACE]]
+            );
+            $searchIndex = $prevWhitespaceTokenIndex;
+        } while (
+            $prevWhitespaceTokenIndex
+            && strpos($tokens[$prevWhitespaceTokenIndex]->getContent(), "\n") === false
+        );
+        $existingIndentation = $prevWhitespaceTokenIndex ?
+            ltrim($tokens[$prevWhitespaceTokenIndex]->getContent(), "\n") :
+            '';
+
+        $indentation = $existingIndentation.$this->whitespacesConfig->getIndent();
+        $endFunctionIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startFunctionIndex);
+        if (!$this->isNewline($tokens[$endFunctionIndex - 1])) {
+            $tokens->insertAt(
+                $endFunctionIndex,
+                new Token([
+                    T_WHITESPACE,
+                    $this->whitespacesConfig->getLineEnding().$existingIndentation,
+                ])
+            );
+            ++$endFunctionIndex;
+        }
+
+        for ($index = $endFunctionIndex - 1; $index > $startFunctionIndex; --$index) {
+            $token = $tokens[$index];
+
+            // skip nested method calls and arrays
+            if ($token->equals(')')) {
+                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index, false);
+                continue;
+            }
+
+            // skip nested arrays
+            if ($token->isGivenKind(CT::T_ARRAY_SQUARE_BRACE_CLOSE)) {
+                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_ARRAY_SQUARE_BRACE, $index, false);
+                continue;
+            }
+
+            if ($token->equals(',')) {
+                $this->fixNewline($tokens, $index, $indentation);
+            }
+        }
+        if (!$this->isNewline($tokens[$startFunctionIndex + 1])) {
```

Comment:
Yes, I did it by adding a new parameter. Is that what you were thinking of? Or did you want something that would override the existing token only if it is a whitespace?